enterprise-suite/install-es.sh

```diff
@@ -64,7 +64,7 @@ function import_credentials() {
 # User overridable variables.
```

Comment:
While we're at it, does it make sense to move this to `enterprise-suite/scripts`?

Reply:
yeah good question.  thoughts @jsravn ?

Reply:
Customers will be using this shell script, that's why I left it here. But we could put it in `scripts`, it's not a big deal I think.

Reply:
Did you end up moving it @dbrinegar?

Reply:
He did not.  I could move it as part of https://github.com/lightbend/helm-charts/pull/100...

Reply:
Created https://github.com/lightbend/helm-charts/pull/105 for moving the `install-es.sh` script.